op-challenger/game/fault/types/types.go

```diff
@@ -111,6 +112,14 @@ type Claim struct {
 	ParentContractIndex int
 }
 
+// Entry creates a hash of the claim.
+func (c *Claim) Entry() common.Hash {
+	gindex := c.ToGIndex().Bytes()
+	valueBytes := c.Value.Bytes()
+	parentContractIndex := big.NewInt(int64(c.ParentContractIndex)).Bytes()
+	return common.Hash(crypto.Keccak256(append(gindex, append(valueBytes, parentContractIndex...)...)))
+}
```

Comment:
I'm not convinced this is a good approach.  This creates a very opaque view of the claim when it would be much clearer to just use the `Claim` itself and have it implement the appropriate comparison interfaces.